src/tests/ftest/dfuse/daos_build.py

```diff
@@ -203,6 +203,9 @@ def run_build_test(self, cache_mode, intercept=False, dfuse_namespace=None):
                 'python3 -m pip install pip --upgrade',
                 'python3 -m pip install -r {}/requirements.txt'.format(build_dir),
                 'scons -C {} --jobs {} --build-deps=only'.format(build_dir, build_jobs),
+                'daos filesystem query {}'.format(mount_dir),
+                'daos filesystem evict {}'.format(build_dir),
+                'daos filesystem query {}'.format(mount_dir),
```

Comment:
Is this a fix or a workaround?

Reply:
Neither, this is to see how long the forget takes and see how effective it is.  Plus check that it doesn't deadlock of course.

Reply:
First results are in, all 3 commands took less than 0.00 seconds so the evict is fast, the first two commands reported 45065 inode in memory, the last one 4951.  I don't think this test should be putting anything other than the build_dir so it would be interesting to see if nodes does drop to 1, if we could run the command in json mode remotely and get the evicted inode number then we could wait for it but I think that's outside the scope of this PR.

Reply:
All tests had exactly the same values which shouldn't be a surprise, the files that aren't being evicted are the venv files which makes sense.